test_driver/test_dsygvdx.F90

```diff
@@ -214,7 +229,11 @@ program main
   w2_d = 0
   ts = wallclock()
   call nvtxStartRange("cuSOLVER",5)
+#ifdef HAVE_CUSOLVERDNDSYGVDX
+  istat = cusolverDnDsygvdx(h, CUSOLVER_EIG_TYPE_1, CUSOLVER_EIG_MODE_VECTOR, CUSOLVER_EIG_RANGE_I, CUBLAS_FILL_MODE_UPPER, N, A2_d, lda, B2_d, lda, 0.D0, 0.D0, il, iu, h_meig, w2_d, work_d, lwork_d, devInfo_d)
+#else
   istat = cusolverDnDsygvd(h, CUSOLVER_EIG_TYPE_1, CUSOLVER_EIG_MODE_VECTOR, CUBLAS_FILL_MODE_UPPER, N, A2_d, lda, B2_d, lda, w2_d, work_d, lwork_d, devInfo_d)
+#endif
   call nvtxEndRange
   te = wallclock()
 
```

Comment:
Can you update the terminal output string after this runs to print `Time for cusolverDnDsygvdx` if the new API is used?

Reply:
done